### ISSUE TITLE
CRITICAL - Could not create main window: No module named 'PyQt5'

### ISSUE THREAD

**Issue by lonnylundsten** [mbari-org/vars-gridview#59]
Updated to the latest and got the following error:

`2023-11-28 10:19:50,640 - vars-gridview - CRITICAL - Could not create main window: No module named 'PyQt5'`

**Comment by kevinsbarnard** [comment]
Looking into this now

**Comment by kevinsbarnard** [comment]
We found a workaround by creating a separate virtual environment and installing gridview fresh:
```bash
virtualenv vars_gridview_env
source vars_gridview_env/bin/activate
pip install vars-gridview==0.5.0
vars-gridview
```

**Comment by lonnylundsten** [comment]
I'm also noticing this in the terminal, although it seems to be loading fine:

```
Exception ignored in: <function SharktopodaClient.__del__ at 0x1696f3920>
Traceback (most recent call last):
  File "/Users/lonny/.pyenv/versions/3.12.0/lib/python3.12/site-packages/sharktopoda_client/client.py", line 581, in __del__
    self.stop_server()
  File "/Users/lonny/.pyenv/versions/3.12.0/lib/python3.12/site-packages/sharktopoda_client/client.py", line 578, in stop_server
    self._udp_server.stop()
  File "/Users/lonny/.pyenv/versions/3.12.0/lib/python3.12/site-packages/sharktopoda_client/udp.py", line 116, in stop
    self._thread.join()
    ^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'join'
```

**Comment by kevinsbarnard** [comment]
@lonnylundsten Thanks for the details! Roger, that was a bug in the sharktpoda-client library. I just pushed a fix for this, so upgrading should do the trick:
```bash
pip install sharktopoda-client==0.4.5
```

**Comment by lonnylundsten** [comment]
Yep, that fixed it!
`pip install sharktopoda-client==0.4.5`

**Comment by lonnylundsten** [comment]
I changed the port numbers and got vars-gridview and sharktopoda talking, but now I'm getting this error:

`MP4 video metadata is missing width or height.` This is for V4464 and video playback with sharktopoda was working yesterday.

**Comment by kevinsbarnard** [comment]
Looks like there was a bug in the Sharktopoda 2 resize calculation, added from #39. Working on it now.

**Comment by kevinsbarnard** [comment]
By the way, this original issue was caused by a bug in `qdarkstyle` 3.2.2. The author released a fix (3.2.3) today that addresses this. Relevant issue: https://github.com/ColinDuquesnoy/QDarkStyleSheet/issues/341

**Comment by lonnylundsten** [comment]
OK> Great. Thanks for sleuthing that and tackling the sharktopoda issue.

**Comment by kevinsbarnard** [comment]
Sure thing! Ok, this should be fixed in v0.5.1 @lonnylundsten.
